extensions/wikia/Chat2/Chat_setup.php

```diff
@@ -191,9 +203,34 @@
 		'chat-private-headline',
 		'chat-user-blocked',
 		'chat-user-allow',
+	],	
+	'dependencies' => [
+		// TODO: load socket.io via RL
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)